numpy/random/bit_generator.pxd

```diff
@@ -14,7 +14,7 @@ cdef class BitGenerator():
 cdef class SeedSequence():
     cdef readonly object entropy
     cdef readonly tuple spawn_key
-    cdef readonly int pool_size
+    cdef readonly uint32_t pool_size
```

Comment:
Is there a reason this (and the other counts in this struct) isn't size_t?

Reply:
`pool_size` is supposed to be tiny (relatively speaking). `n_children_spawned` could be largish, but should be capped at `uint32_t` (as it turns into a `uint32_t` element in the `spawn_key`).

Reply:
The docstring for the `pool_size` attribute says "Size of the pooled entropy to store. Default is 4 to give a 128-bit entropy pool. 8 (for 256 bits) is another reasonable choice if working with larger PRNGs, but there is very little to be gained by selecting another value."  So 32 bits is far more than necessary, but I guess it is harmless to leave it that size.

>  `n_children_spawned` could be largish, but should be capped at `uint32_t` (as it turns into a `uint32_t` element in the `spawn_key`).

OK, I changed `n_children_spawned` to `uint32_t` in the changes that I just pushed.